srcpkgs/docker-engine/template

```diff
@@ -0,0 +1,60 @@
+# Template file for 'docker-engine'
+pkgname=docker-engine
+version=20.10.5
+revision=1
+archs="aarch64* armv[67]l* x86_64* ppc64le*"
+create_wrksrc=yes
+build_style=go
+go_import_path="github.com/docker/docker"
+hostmakedepends="pkg-config tar"
+makedepends="libbtrfs-devel device-mapper-devel libseccomp-devel"
+depends="containerd runc iptables xz git"
```

Comment:
Here too, is `git` really a hard dependency of `moby`? Also drop the `runc` since `containerd` pulls it in.

Reply:
This is my mistake I believe, git is a hard dependency of the docker-cli, it uses it to pull packages and handle version tagging, this must have been left in from the split.

Reply:
Fixed.